intergov/processors/subscription_handler/__init__.py

```diff
@@ -0,0 +1,44 @@
+import logging
```

Comment:
Does it work with sentry?

Reply:
probably not, do you mean that I should import `intergov.loggers` to be able to make it work?

Reply:
I had some weird problems with logging using `import logging`, so yes please, use our logging as in other places